### DISCUSSION TITLE
[Analytics] Disable purchase events and revenue [iOS]

### DISCUSSION THREAD

**Discussion by nijakobius** [capawesome-team/capacitor-firebase#359] [Q&A]
I am using RevenueCat and want Google Analytics to receive purchase events and revenue ONLY from the Firebase x RevenueCat integration.

To avoid double collection of such data, I must disable that it is automatically collected by the Capacitor Firebase Analytics plugin.

For Android, there is a setting for this under Firebase > Project settings > Integrations > Unlink Google Play.

For iOS there is no such setting and revenue and purchase events are always collected and shown in Google Analytics (see attached screenshot). Maybe there should be an option within the analytics plugin to disable the collection/transmission of such data? :) 

<img width="1587" alt="Screenshot 2023-04-09 at 13 24 32" src="https://user-images.githubusercontent.com/37240226/230770581-23dec217-3bbd-47c7-8cfe-04c96ee7434d.png">

**Comment by robingenz** [comment]
Do you have a reference to the documentation where it says what needs to be implemented?
I know only the section [`Disable Analytics data collection`](https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#disable_data_collection).
This functionality is already implemented with [`setEnabled(...)`](https://github.com/capawesome-team/capacitor-firebase/tree/main/packages/analytics#setenabled).
Have you asked RevenueCat support how other customers solve this problem using their integration?
